src/views/auth/ChoosePlace.js

```diff
@@ -24,8 +24,14 @@ class ChoosePlace extends React.Component {
     this.gameId = sessionManager.getGameId();
     this.channels = [
       createChannel(`/topic/game/${this.gameId}/colors`, (msg) => this.handleChoosePlaceMessage(msg)),
+      createChannel(`/topic/game/${this.gameId}/game-end`,(msg)=> this.handlePlayerDisconnection(msg))
     ]
   }
+  handlePlayerDisconnection(msg) {
```

Comment:
mmmh seems to be the same code as in the game handlePlayerDisconnection(), I think it's better to encapsulate that logic to the InGameView.js